Modules/_testcapimodule.c

```diff
@@ -3036,15 +3036,19 @@ static PyObject *
 check_pyimport_addmodule(PyObject *self, PyObject *args)
 {
     const char *name;
-    if (!PyArg_ParseTuple(args, "s", &name)) {
+    int is_new;
+    if (!PyArg_ParseTuple(args, "si", &name, &is_new)) {
         return NULL;
     }
+    // name must be the name of a module which is already in sys.modules
```

Comment:
This comment is wrong for the `is_new=True` case.
If you've read the function's docs, the test code should be clear even without a similar comment, so I suggest removing it.
```suggestion
```